### PR TITLE
Feat/terce output

### PR DESCRIPTION
**Pull request by EESergey**
Please do not delete this branch, I will delete it myself after changing the test configuration.

### CODE REVIEW

**Review by carpawell — COMMENTED** [review]
> Please do not delete this branch

So it is not expected to be merged? Why is it a PR (even opened, not a draft)?

File: scenarios/grpc.js [hunk 1]
```diff
@@ -163,3 +163,34 @@ export function uuidv4() {
...
+    const c4 = ~~(data.metrics.neofs_obj_get_duration && data.metrics.neofs_obj_get_duration.values.avg || '0');
+    const c5 = ~~(data.metrics.neofs_obj_get_duration && data.metrics.neofs_obj_get_duration.values["p(95)"] || '0');
+    const e6 = (data.metrics.neofs_obj_put_fails && data.metrics.neofs_obj_put_fails.values.rate || '0');
+    const c6 = ~~((data.metrics.neofs_obj_put_total && data.metrics.neofs_obj_put_total.values.rate || '0') - e6);
+    const c7 = ~~(data.metrics.neofs_obj_put_duration && data.metrics.neofs_obj_put_duration.values.avg || '0');
+    const c8 = ~~(data.metrics.neofs_obj_put_duration && data.metrics.neofs_obj_put_duration.values["p(95)"] || '0');
+    const e9 = (data.metrics.neofs_obj_delete_fails && data.metrics.neofs_obj_delete_fails.values.rate || '0');
+    const c9 = ~~((data.metrics.neofs_obj_delete_total && data.metrics.neofs_obj_delete_total.values.rate || '0') - e9);
+    const c10 = ~~(data.metrics.neofs_obj_delete_duration && data.metrics.neofs_obj_delete_duration.values.avg || '0');
+    const c11 = ~~(data.metrics.neofs_obj_delete_duration && data.metrics.neofs_obj_delete_duration.values["p(95)"] || '0');
```
Comment (smallhive) [inline scenarios/grpc.js:189]: Do these names mean something special?

**Comment by EESergey** [comment]
> > Please do not delete this branch
> 
> So it is not expected to be merged? Why is it a PR (even opened, not a draft)?

This is not a draft, this branch is currently used for testing.

File: scenarios/grpc.js (hunk 1 above)
Comment (EESergey) [reply scenarios/grpc.js:189]: No, these names are chosen for convenience, they can be changed to something more meaningful.

Comment (EESergey) [reply scenarios/grpc.js:189]: But if I change the names, the output line will be too long.

**Comment by carpawell** [comment]
> This is not a draft

I meant that it is an open (not draft) PR with a review request, such PRs are usually expected to be merged, so I am confused.

File: scenarios/grpc.js (hunk 1 above)
Comment (smallhive) [reply scenarios/grpc.js:189]: The output line is already too long, but I do not insist

**Review by roman-khimov — APPROVED** [review]
Not sure it's good long-term (looks like we're still misusing k6 in some ways), but if it helps us now then we can have it and improve upon it.

**Pull request merged**